xml/favicons.xml

```diff
@@ -2346,6 +2346,14 @@
     <param pos="0" name="service.product" value="R1Soft Server Backup Manager"/>
   </fingerprint>
 
+
+  <fingerprint pattern="^cc21dae362a981ea9ce93138cb855e66$">
```

Comment:
Please add the hashes for the other two favicon files `favicon-32x32.png` and `favicon-16x16.png` and include an XML comment above the `example` that documents the filename and software version. I know there are not many fingerprints including these details at the moment, but we think it will be helpful in the future when updating an existing fingerprint.

For example:
```
<!-- favicon.ico from version X.Y.Z -->
<example>cc21dae362a981ea9ce93138cb855e66</example>
<!-- favicon-32x32.png from version X.Y.Z -->
<example>TBD</example>
<!-- favicon-16x16.png from version X.Y.Z -->
<example>TBD</example>
```

https://github.com/Fallenbagel/jellyseerr/tree/develop/public